Makefile

```diff
@@ -4,27 +4,31 @@ PREFIX ?= $(DESTDIR)/usr
 BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 UNITDIR ?= $(PREFIX)/lib/systemd/system
-DSSLIBDIR ?= $(PREFIX)/lib/docker-storage-setup
+CSSLIBDIR ?= $(PREFIX)/lib/container-storage-setup
 SYSCONFDIR ?= $(DESTDIR)/etc/sysconfig
 
 .PHONY: clean
 clean:
 	-rm -rf *~ \#* .#*
 
 .PHONY: install
-install: install-docker
+install: install-docker install-core
 
 .PHONY: install-docker
-install-docker: install-core
-	install -D -m 644 ${DOCKER}-storage-setup.service ${UNITDIR}/${DOCKER}-storage-setup.service
+install-docker:
```

Comment:
Given we are planning to package it separately, I think docker package will have to do all this manually anyway and not rely on these targets.  Even if docker were to use it, they would like to
get core bits as well as comaptibility bits. Can't see how compatibility bits alone are helpful to anybody.